br2idosom2d01/board/wirelesstag/idosom2d01/linux.config

```diff
@@ -302,6 +303,13 @@ CONFIG_SQUASHFS_LZO=y
 CONFIG_SQUASHFS_XZ=y
 CONFIG_SQUASHFS_ZSTD=y
 CONFIG_NFS_FS=y
+CONFIG_UBIFS_FS=y
```

Comment:
I wonder if we should make this into a config fragment so people that want it can enable it? The problem is that config fragments then stop you from saving a defconfig.

Reply:
I mean people can change to what they want to by default, this is only to have supported UBIFS as a lot more is enabled by default on kernel that is not used by many IMHO but it's up to you if you want this to be in fragment or not

Reply:
Let me have a think about this. I'm working on getting lvgl to build here too so making UIs is easy but that would also mean turning on all of the display options that I don't want for everything.

I think the idea should be that you can create a branch for a project with the options you need for that and pull master from here to get updates without breaking it.

Reply:
No worries will do that and also :crossed_fingers:  with lvgl